hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/MemStoreCompactor.java

```diff
@@ -46,7 +46,7 @@
 
   // Option for external setting of the compacted structure (SkipList, CellArray, etc.)
   static final String COMPACTING_MEMSTORE_TYPE_KEY = "hbase.hregion.compacting.memstore.type";
-  static final int COMPACTING_MEMSTORE_TYPE_DEFAULT = 2;  // COMPACT_TO_ARRAY_MAP as default
+  static final int COMPACTING_MEMSTORE_TYPE_DEFAULT = 2;  // COMPACT_DATA_TO_ARRAY_MAP as default
```

Comment:
Is it possible to change this to String; using integers is meaningless and may confuse the users (so far it's just us but anoop at el may also play with these when we get to chunk map)